Mathlib/Analysis/SpecialFunctions/Complex/CircleAddChar.lean

```diff
@@ -83,4 +90,17 @@ lemma isPrimitive_stdAddChar (N : ℕ) [NeZero N] :
   refine AddChar.zmod_char_primitive_of_eq_one_only_at_zero _ _ (fun t ht ↦ ?_)
   rwa [← (stdAddChar (N := N)).map_zero_eq_one, injective_stdAddChar.eq_iff] at ht
 
+/-- `ZMod.toCircle` as an `AddChar` into `rootsOfUnity n Circle`. -/
+noncomputable def expHom (n : ℕ) [NeZero n] : AddChar (ZMod n) (rootsOfUnity n Circle) where
```

Comment:
The name should mention roots of unity somehow; maybe `rootsOfUnityAddChar`?

Probably it is also useful to have a rfl-lemma accompanying the definition, maybe
```
@[simp] lemma rootsOfUnityAddChar_val (n : ℕ) [NeZero n] (x : ZMod n) : 
    (rootsOfUnityAddChar n x).val = toCircle x := by
  rfl
```